webextensions/manifest/options_page.json

```diff
@@ -12,7 +12,9 @@
               "version_added": "14"
             },
             "firefox": {
-              "version_added": false
+              "alternative_name": "options_ui.page",
+              "version_added": "126",
+              "notes": "An alias to <code>options_ui.page</code>. Ignored when <code>options_ui.page</code> is set."
```

Comment:
nit: may be worth to callout in the note that options_page is techically an alias for `<code>"options_ui": { page: "...", "open_in_tab": true }</code>`.

Reply:
I'd omit alternative_name and the notes. Just version_added suffices. options_ui.page is an alternative in all browsers, it is not specific to any browser so it doesn't need a bcd note.

Reply:
:+1: Fine by me too, let's keep this simple.